doc/classes/AudioEffectChorus.xml

```diff
@@ -14,78 +14,90 @@
 			<return type="float" />
 			<param index="0" name="voice_idx" type="int" />
 			<description>
+				Gets the volume of a specific voice in decibels.
 			</description>
 		</method>
 		<method name="get_voice_pan" qualifiers="const">
 			<return type="float" />
 			<param index="0" name="voice_idx" type="int" />
 			<description>
+				Gets the pan level of a specific voice.
 			</description>
 		</method>
 		<method name="get_voice_rate_hz" qualifiers="const">
 			<return type="float" />
 			<param index="0" name="voice_idx" type="int" />
 			<description>
+				Gets the filter rate frequency of a specific voice.
 			</description>
 		</method>
 		<method name="set_voice_cutoff_hz">
 			<return type="void" />
 			<param index="0" name="voice_idx" type="int" />
 			<param index="1" name="cutoff_hz" type="float" />
 			<description>
+				Sets the cutoff frequency of a specific voice in hertz (number of samples per second). Cutoff frequency refers to the top-most frequency beyond which the signal is blocked. This can be used to create a low pass filter to simulate a "muffled" sound. The recommended range is [code]0[/code] to [code]20500[/code].
 			</description>
 		</method>
 		<method name="set_voice_delay_ms">
 			<return type="void" />
 			<param index="0" name="voice_idx" type="int" />
 			<param index="1" name="delay_ms" type="float" />
 			<description>
+				Sets the delay of a specific voice in milliseconds. Longer delay creates more audible separation between the voices. The recommended range is [code]0[/code] to [code]50[/code]. [b]Note: [/b] the maximum recommended value can be exceeded to create longer more noticeable separation.
 			</description>
 		</method>
 		<method name="set_voice_depth_ms">
 			<return type="void" />
 			<param index="0" name="voice_idx" type="int" />
 			<param index="1" name="depth_ms" type="float" />
 			<description>
+				Sets the depth of a specific voice filter in milliseconds. Increasing the audio depth increases the intensity of a sound. The recommended range is [code]0[/code] to [code]20[/code]. [b]Note:[/b] The maximum value can be exceeded to create distortions.
 			</description>
 		</method>
 		<method name="set_voice_level_db">
 			<return type="void" />
 			<param index="0" name="voice_idx" type="int" />
 			<param index="1" name="level_db" type="float" />
 			<description>
+				Sets the volume of a specific voice in decibels. This is the base volume before attenuation. The recommended range is [code]-60[/code] to [code]24[/code].
 			</description>
 		</method>
 		<method name="set_voice_pan">
 			<return type="void" />
 			<param index="0" name="voice_idx" type="int" />
 			<param index="1" name="pan" type="float" />
 			<description>
+				Sets the pan level of a specific voice. Values range from [code]-1.0[/code] (left ear) to [code]1.0[/code] (right ear). [b]Note:[/b] This effect does not work with stereo sounds, but mono can be forced in import settings.
```

Comment:
```suggestion
				Sets the pan level of a specific voice. Values range from [code]-1.0[/code] (left ear) to [code]1.0[/code] (right ear).
				[b]Note:[/b] This effect does not work with stereo sounds, but mono can be forced in import settings.
```
But is the note true? What does it mean? Is it about an AudioStream's import settings?

Reply:
I have tested it to be true. I imported a stereo sound and applied the pan effect, and there was no audible change until I forced mono import. I think the channel separation within the audio file must override any stereo separation we add through the pan effect.

Reply:
It still needs to be more explicit, especially because only **AudioStreamWAV** can toggle stereo this way.
```suggestion
				Sets the pan level of a specific voice. Values range from [code]-1.0[/code] (left ear) to [code]1.0[/code] (right ear).
				[b]Note:[/b] The chorus effect does not work with stereo audio. For [AudioStreamWAV] samples, mono can be forced by setting [member AudioStreamWAV.stereo] to [code]false[/code].
```

Reply:
I didn't notice that, you're right about WAV being the only file type that can have mono forced in the editor. It looks like OGG and MP3 will only work with pan if the file itself is already mono. Thanks for the fix.